src/public/components/text-input.js

```diff
@@ -0,0 +1,7 @@
+import K, * as U from "karet.util"
+import React     from "karet"
+
+export const TextInput = ( { value } ) =>
+  <input type="text"
+         value={ value }
+         onChange={ U.getProps( { value } ) }/>
```

Comment:
This is a very helpful example that answers some questions I had. I had considered adding `...props` to some of the components - I had it on `pretty-stringify`, then decided to remove it. The notes about `mount` and `onChange` are new and helpful information for me, as well.